src/library/LibraryWindow.vala

```diff
@@ -144,8 +144,8 @@ public class LibraryWindow : AppWindow {
 
     construct {
         set_default_size (
-            window_settings.get_int ("library-width"),
-            window_settings.get_int ("library-height")
+            window_settings.get ("library-position", "(ii)", out window_x, out window_y);
+            window_settings.get ("library-size", "(ii)", out width, out height);
```

Comment:
You need to rewrite this because  `settings.get ()` does not return a value so cannot be used as parameter for `set_default_size ()`.

This should work  (I havent checked it)
```
int width, height;
window_settings.get ("library-size", "(ii)", out width, out height);
set_default_size (width, height);
```